### PR TITLE
Install CMake in Merlin base image (instead of copying from build)

### PR DESCRIPTION
**Pull request by karlhigley**
Since the CMake install is a bit more involved than just copying the executable, let's just install it in the base container directly.

### CODE REVIEW

**Comment by karlhigley** [comment]
rerun tests

**Comment by karlhigley** [comment]
rerun tests

File: docker/dockerfile.merlin [hunk 1]
```diff
@@ -265,8 +265,9 @@ RUN apt update -y --fix-missing && \
 
 RUN ln -s /usr/bin/python3 /usr/bin/python
 
-# CMake
-COPY --chown=1000:1000 --from=build /usr/local/bin/cmake /usr/local/bin/cmake
+# Install cmake (used by downstream images)
```
Comment (oliverholworthy) [inline docker/dockerfile.merlin:268]: There are a couple of lines below this where we're copying cmake from `lib` and `bin`. Might be able to remove those if this is a replacement for those.

**Review by oliverholworthy — APPROVED** [review]

File: docker/dockerfile.merlin (hunk 1 above)
Comment (karlhigley) [reply docker/dockerfile.merlin:268]: It's actually supposed to be a replacement for installing CMake in the downstream images, which was happening over the top of those copies. I think we can remove them though

Comment (karlhigley) [reply docker/dockerfile.merlin]: Added a commit to address this

**Pull request merged**